File: Domain/Pizza.cs
using Domain.Enums;
using System;

namespace Domain
{
    public class Pizza
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public bool IsAvailable { get; set; }
        public decimal PriceForSmall { get; set; }
        public decimal PriceForLarge { get; set; }
        public decimal PriceForXXL { get; set; }
        public PizzaCategories PizzaCategory { get; set; }
    }
}

File: client-app/src/app/modules/pizza.ts
import { pizzaCategory } from "./pizzaCategory";

export interface IPizza{
    id:string;
    name:string;
    description:string;
    isAvailable:boolean;
    priceForSmall:number;
    priceForLarge:number;
    priceForXXL:number;
    pizzaCategory:pizzaCategory;
}